pkgs/test_api/lib/src/backend/live_test_controller.dart

```diff
@@ -186,9 +193,15 @@ class LiveTestController {
     return liveTest.onComplete;
   }
 
+  /// Returns a future that completes when the test is complete.
+  ///
+  /// We also wait for the state to transition to Status.complete.
+  Future<void> get onComplete =>
```

Comment:
These changes should be outlined in the `CHANGELOG` and we should bump the `pubspec` version.

Reply:
Done.